### PR TITLE
cli :Implement events command

### PR DESCRIPTION
**Pull request by jshachm**
Events cli display container events such as cpu,memory, and IO usage statistics.

By now OOM notifications are not fully supproted.

Fixes : #186

### CODE REVIEW

**Comment by jodh-intel** [comment]
Could you raise issues for OOM support (and Intel RDT ;) and add the `limitation` label for both please?

The CI is failing due to a few lint errors which need fixing. Once done,

lgtm

**Comment by jshachm** [comment]
@jodh-intel  thx. Working on fix lint error. Issues will be raised later ~

**Comment by jshachm** [comment]
@jodh-intel 
issues are raised and lint errors have been fixed. 

But I think I still need to add tests file like `events_test.go` and some other. 

Will update these later~

**Comment by jodh-intel** [comment]
Thanks @jshachm!

**Comment by jshachm** [comment]
working on fixing codecov and add some other tests.Once get it done, I think it should be ready to review.

**Comment by jshachm** [comment]
@bergwolf @sboeuf @jodh-intel 

PTAL

tests have been added~

File: virtcontainers/interfaces.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ type VC interface {
 	KillContainer(sandboxID, containerID string, signal syscall.Signal, all bool) error
 	StartContainer(sandboxID, containerID string) (VCContainer, error)
 	StatusContainer(sandboxID, containerID string) (ContainerStatus, error)
+	StatsContainer(sandboxID, containerID string) (ContainerStats, error)
```
Comment (bergwolf) [inline virtcontainers/interfaces.go]: Can you please make this part of the VCSandbox interface and let VC.StatsContainer be a wrapper of it, like we did for the other sandbox APIs?

Comment (jshachm) [reply virtcontainers/interfaces.go]: @bergwolf  Got it~ On the way to fix it

**Comment by jshachm** [comment]
@bergwolf 
Interface changes done.
PTAL
But CI for ubuntu-17.10 failes randomly. So strange....

**Comment by bergwolf** [comment]
LGTM. Thanks @jshachm!

**Comment by jshachm** [comment]
@bergwolf  thanks 
OK, let's begin fighting with the strange CI....

**Comment by jshachm** [comment]
@jodh-intel @bergwolf   CI is happy now. Ready to merge~ thx

**Pull request merged**